assets/js/barChart.js

```diff
@@ -149,8 +184,10 @@ class BarChart {
 
         // Transform license count into array of objects with license name & count
         let chartData = $.map(count, function(val, key) {
+            let trimmedName = BarChart.trimName(key);
```

Comment:
This is short enough now to where you can just move it into the object if you want:

`trimmedName: BarChart.trimName(key);`